Cli docs

gauntface opened the pull request:
R: @jeffposnick @addyosmani @jpmedley 

This is necessarily done, but I'm hoping we can land some content and add / re-organise in future PR's.

Let me know if you think this is a stupid idea.

jpmedley commented on docs/getting-started/index.md:
```diff
@@ -0,0 +1,137 @@
+## Generating a Service Worker
+
+To generate a service worker for your project just run
+the CLI with the 'generate:sw' command.
```
Do I run it from the root of my project, or somewhere else?

gauntface replied:
root of your project.

jpmedley commented on docs/getting-started/index.md:
```diff
@@ -0,0 +1,137 @@
+It allows you to generate the service worker  
+programmatically.
+
+[Learn More About sw-build Here](.)
```
@jeffposnick You wrote some build instructions, did you? If the need conversion from Docs, send them over to me.

gauntface replied:
I will link up Jeff's docs on here once the PR is landed.

jeffposnick replied:
I don't think think it's been published to `gh-pages` yet, but it's landed at https://github.com/GoogleChrome/sw-helpers/blob/master/docs/recipes/build/rollup.md

jpmedley commented on docs/getting-started/index.md:
```diff
@@ -0,0 +1,137 @@
+Both `sw-cli` and `sw-build` can be used to produce a
+list of assets in your web app with the revision
+information which can be used for precaching. We have
+[recipes on how to do this](.).
```
Borken link again?

gauntface replied:
Non existent link as waiting on PR.

jeffposnick replied:
Ditto re: https://github.com/GoogleChrome/sw-helpers/blob/master/docs/recipes/build/rollup.md

jpmedley commented on docs/getting-started/index.md:
```diff
@@ -0,0 +1,137 @@
+you can define custom routes and add anything else you
+desire to your service worker.
+
+[Learn More About sw-lib Here](.)
```
And another.

gauntface replied:
Non existent link as waiting on PR for your content..

gauntface commented:
@jeffposnick @jpmedley 

Fixed to valid links for everything - can change them when we get more relevant content.

I also added an npm run script "npm run serve" to serve the doc site locally - the recipes needed some front matter to be included in the nav - although I'm not 100% confused that travis is building master branch but will see when this lands.
